app/scripts/lib/app-start.js

```diff
@@ -372,6 +373,11 @@ define(function (require, exports, module) {
             relier: this._relier,
             window: this._window
           });
+        } else if (this._isFxDesktopV3()) {
+          this._authenticationBroker = new FxDesktopV3AuthenticationBroker({
```

Comment:
Nothing to do with this change, but one day perhaps we should consider doing something about this function. The repetition could be nicely eliminated by looping through a data structure that describes each broker.